README.md

```diff
@@ -62,20 +62,15 @@ and `localhost/backend/admin` and ensure everything is working as expected.
 When building images for production, it's recommended to add an additional .env-file called `.env.prod`. Do not add this
 file to version control. Reference the .env-file with the `--env-file` flag when building the images for production.
 
-### Pushing docker images
+### Deploying on production
 
-Push the docker images to a <ins>**private**</ins> repository. This is essential, to prevent secret in your .env-files
-from being exposed. You can push the images using:
+Make sure your latest changes have been pushed. `ssh` into the production instance. `cd` to `~/roodjongeren`.
+This is where the docker compose file is, as well as the .env file. `cd roodjongeren` to open the cloned repository.
+Pull the latest changes from GitHub via `sudo git pull`.
```

Comment:
Voor `git pull` hebben we volgens mij geen sudo nodig

Reply:
Jawel; probeer maar